config/settings.py

```diff
@@ -256,3 +258,7 @@
 DEFAULT_REQUEST_TIMEOUT = get_env(
     "OSIDB_DEFAULT_REQUEST_TIMEOUT", default="30", is_int=True
 )
+
+CORS_ALLOWED_ORIGINS = [
+  "https://ui.osim-ui--runtime-int.apps.int.spoke.preprod.us-east-1.aws.paas.redhat.com"
```

Comment:
This expose an internal url, I'd like to recommend using get_env("WEBUI_URL") or any other name, include it in your `.env` file and list it on `docs/developer/DEVELOP.md`. I can add it on the ops project and deploy it on stage environment.

Reply:
To be honest I'm not sure what else we would like to plug on OSIDB, but I believe one variable that represents the frontend and change with the environment would be enough, but I'm ok if we decide to go for a more robust solution that allows multiple URLs.

Reply:
We will have, in the very near future, both a staging and prod url that correlates with the staging and prod MP+ clusters. I can make an adjustment to settings.py to read that env as an array.

Reply:
I would also highly suggest setting up at least the stage CNAME and going with that instead of the canonical URL, but in general we can make those changes on-the-fly without requiring a new OSIDB release if we use the JSON array as env var (see https://github.com/RedHatProductSecurity/osidb/blob/master/osidb/helpers.py#L24)